internal/configtxgen/encoder/encoder_test.go

```diff
@@ -1167,7 +1154,7 @@ var _ = Describe("Encoder", func() {
 				Expect(configUpdate.WriteSet.Groups["Application"].Groups["Org1"].Version).To(Equal(uint64(1)))
 				Expect(configUpdate.WriteSet.Groups["Application"].Groups["Org1"].Values["AnchorPeers"]).NotTo(BeNil())
 				Expect(configUpdate.WriteSet.Groups["Application"].Groups["Org2"].Version).To(Equal(uint64(0)))
-				Expect(configUpdate.WriteSet.Groups["Orderer"].Values["ConsensusType"].Version).To(Equal(uint64(1)))
```

Comment:
Can you explain what part of the change makes this assertion change? 

Is it this code section that was removed?

```
		addValue(ordererGroup, channelconfig.KafkaBrokersValue(conf.Kafka.Brokers), channelconfig.AdminsPolicyKey)
```

Reply:
No, it is line 1127 (old) => 1114 (new) in this file `encoder_test.go`.

![image](https://user-images.githubusercontent.com/15885305/188439021-54848bfc-b2fc-49ff-8605-aafca7830d25.png)

The test changed the consensus type from solo to kafka, but now it remains solo, so there is no change detected in the orderer group.

Reply:
I found a better way to change the test - change the BatchTimeout, it achieves the same effect.